rootfs/etc/nginx/lua/balancer/static_balancers.lua

```diff
@@ -0,0 +1,79 @@
+local util = require('util')
+local split = require('util.split')
+local ngx_upstream = require("ngx.upstream")
+local implementations = require('balancer.implementations')
+
+local backends = {}
+local balancers = {}
+
+local DEFAULT_LB_ALG = "ewma"
+
+local _M = {}
+
+local function marshal_endpoint(endpoint)
```

Comment:
Maybe `parse_endpoint(server)`? When you say marshal I think of encoding an object into a text blob.